### ISSUE TITLE
DLP in wiki

### ISSUE THREAD

**Issue by StefanCohen** [j7126/OctoPrint-Dashboard#328]
The wiki still mentions DLP as a requirement. https://github.com/j7126/OctoPrint-Dashboard/wiki/DisplayLayerProgress Is that intentional or can it be removed?

**Comment by j7126** [comment]
This is correct those are not relevant since 1.19.x, they have been removed.
